src/System.Runtime.Serialization.Xml/tests/S.P.DCS.Tests.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Xml;
+using Xunit;
+
+public static class SystemPrivateDataContractSerializationTest
+{
+    [Fact]
+    public static void Xml_ValueHandle_TryReadChars_Double_Wide_Chars_Encoded()
+    {
+        // The test is to verify the fix made for https://github.com/dotnet/corefx/issues/5352.
```

Comment:
Source control check in comment is the best place to link to bug numbers. It's much helpful for code comment to descript the related scenarios and issues.